.goreleaser.yml

```diff
@@ -43,6 +50,12 @@ brews:
     repository:
       owner: cirruslabs
       name: homebrew-cli
+    license: AGPL-3.0-only
+    install: |
+      bin.install "cirrus"
+      bash_completion.install "completion/bash_completion" => "cirrus"
+      zsh_completion.install "completion/zsh_completion" => "_cirrus"
+      fish_completion.install "completion/fish_completion" => "cirrus.fish"
```

Comment:
Will `make_completion` be automatically called to generate the files?

Reply:
No, `make_completion` is simply a shortcut to populate the `completion` directory.

I can imagine you'd want to run `make_completion` on all PRs to ensure any changes to the CLI interface will require rerunning of `make_completion`.